Classes/StenoSignal.sc

```diff
@@ -127,7 +179,14 @@ StenoSignal {
 	writeToBus {
 		outputSignals !? {
 			outputSignals.keep(numChannels);
```

Comment:
this line does nothing. Maybe you wanted to write something like:

`signals = outputSignals.keep(numChannels);` 

and reuse further down.